utils/calendrical_calculations/src/helpers.rs

```diff
@@ -294,7 +294,8 @@ pub enum I32CastError {
 impl core::error::Error for I32CastError {}
 
 impl I32CastError {
-    pub(crate) const fn saturate(self) -> i32 {
+    /// Recovers the value saturated to `i32:::MIN..=i32::MAX`.
+    pub const fn saturate(self) -> i32 {
```

Comment:
issue: new public API naming.

I think `saturate` is fine. Maybe `saturated_i32`?